addons/l10n_id/data/account_tax_template_data.xml

```diff
@@ -149,4 +149,60 @@
             }),
         ]"/>
     </record>
+    <record id="tax_ST3" model="account.tax.template">
+        <field name="description">ST3</field>
+        <field name="chart_template_id" ref="l10n_id_chart"/>
+        <field name="type_tax_use">sale</field>
+        <field name="name">12%</field>
+        <field name="amount_type">percent</field>
+        <field name="amount">12.0</field>
+        <field name="invoice_repartition_line_ids" eval="[(5, 0, 0),
+            (0,0, {
+                'repartition_type': 'base',
```

Comment:
If it's the new rate for the standard tax, shouldn't we archive the previous one so new db don't get bloated by it.

Also, I in the following version, if it's the new rate for standard, don't forget to replace the old tax by the new in the `template.py`file.

Reply:
It would be good but new db still need the 11% tax until January.
We may as well keep both for a few weeks until we're sure that 11% is no longer needed at all and then archive it.

> Also, I in the following version, if it's the new rate for standard, don't forget to replace the old tax by the new in the template.pyfile

Yep, I actually wanted to change it here too but I guess the setting wasn't precised yet.

Reply:
> Yep, I actually wanted to change it here too but I guess the setting wasn't precised yet.

so, you plan to prepare a PR to merge on January 1st? :thinking:

Reply:
> > Yep, I actually wanted to change it here too but I guess the setting wasn't precised yet.
> 
> so, you plan to prepare a PR to merge on January 1st? 🤔

Nah, my idea was that it was less impacting for the user to have the wrong default tax for a week or two than to have a tax they need being archived by default and hard to find.
But in any case, as of today, we don't really have a perfect solution to handle such use cases. So we may as well not do anything now and issue another pr in 2-3 weeks to update the data/archive the tax by default. I already added a activity for myself on the task to not forget.

Reply:
> I suppose even during the beginning of 2025 you still want to be able to use the 11% tax when encoding invoices from 2024, anyway. So I'd rather let the user archive it himself

I think new users starting from a fresh DB and fresh accounting doesn't need to do (sale) invoice in the past.

It's not the same for bill ofc.

All in all, it doesn't really matter, as I told Nicolas, I could have r+ed in the current state.